build/tasks/esbuild.js

```diff
@@ -0,0 +1,31 @@
+const { build } = require('esbuild');
+const path = require('path');
+
+module.exports = function(grunt) {
+	grunt.registerMultiTask(
+		'esbuild',
+		'Task to run the esbuild javascript bundler',
+		function() {
+			const files = grunt.task.current.data.files;
+
+			files.forEach(file => {
+				const src = Array.isArray(file.src) ? file.src : [file.src];
+				const dest = file.dest;
+
+				src.forEach(entry => {
+					const name = path.basename(entry);
+					if (file.cwd) {
+						entry = path.join(file.cwd, entry);
+					}
+
+					build({
+						entryPoints: [entry],
+						outfile: path.join(dest, name),
+						minify: false,
+						bundle: true
+					}).catch(() => process.exit(1));
```

Comment:
IMO we should output an error here, not just fail silently.

Reply:
If Grunt handles this and outputs a useful error, then ignore this. I don't know how the Grunt architecture works.

Reply:
I think esbuild does that for us? Their docs just exit https://github.com/evanw/esbuild/blob/master/docs/js-api.md#node-api. I had an issue with the config at one point and they messaged the error, so I assume a problem with the build will as well?

Reply:
That looks like example code, not necessarily what should be used in an actual build process.

Have you seen what happens when an exception is raised here?

Reply:
Looks like it outputs the error:

![image](https://user-images.githubusercontent.com/2433219/96758852-8d33c580-1394-11eb-909c-189ac950b64a.png)

Reply:
Ah, perfect! Thanks for looking into this.